shops/templates/shops/weekly_import_shopdetail.html

```diff
@@ -28,7 +28,7 @@ <h1>Sales details</h1>
 
         {% endfor %}
     </table>
-    
+     
```

Comment:
You've now got a lot of duplicated HTML. You should be able to tidy this up using {% include ...